### PR TITLE
Pin Python packages

### PR DESCRIPTION
**Pull request by kumoy**

### CODE REVIEW

**Comment by kbalk** [comment]
It would be nice if there was still an install target so that if you didn't use docker, then you could still install these packages on your system.  That goes for the non-Python packages, too.  It's really convenient for setting up a new system.

If you decide you still want to get rid of the install target, then I believe the README.md needs to be updated?

**Comment by kumoy** [comment]
@lorengordon what do you think? 
Also...something is not liked by Travis :(

File: Dockerfile [hunk 1]
```diff
@@ -28,6 +35,7 @@ RUN apt-get update -y && apt-get install -y \
     make \
     && ln -s /opt/bats/bin/bats /usr/local/bin/bats \
     && make -C /${PROJECT_NAME} install \
```
Comment (lorengordon) [inline Dockerfile:37]: Now that all the tools are being installed through the Dockerfile, this line calling the `install` target should be removed. That will fix the travis-ci failure.

**Comment by kbalk** [comment]
I'll just create an alternative tool for myself then to do the installs.

File: Dockerfile [hunk 2]
```diff
@@ -27,7 +34,7 @@ RUN apt-get update -y && apt-get install -y \
     unzip \
     make \
     && ln -s /opt/bats/bin/bats /usr/local/bin/bats \
-    && make -C /${PROJECT_NAME} install \
+    && pip -m install --no-cache-dir -r /${PROJECT_NAME}/requirements.txt \
```
Comment (lorengordon) [inline Dockerfile:37]:
```suggestion
    && python -m pip install --no-cache-dir -r /${PROJECT_NAME}/requirements.txt \
```

File: requirements.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+black==20.8b1
```
Comment (lorengordon) [inline requirements.txt:1]: Lets order this list and add an empty line between each package to try to help git better detect conflicts.

For that matter, we can add a comment to the top of the file...

```suggestion
# Keep this list in alphabetical order, with a single blank line between each package
black==20.8b1
```

File: Makefile [hunk 4]
```diff
@@ -398,6 +398,4 @@ project/validate:
 	[ "$$(ls -A $(PWD))" ] || (echo "Project root folder is empty. Please confirm docker has been configured with the correct permissions" && exit 1)
 	@ echo "[$@]: Target test folder validation successful"
 
-install: black/install pylint/install pylint-pytest/install pydocstyle/install yamllint/install cfn-lint/install bumpversion/install
```
Comment (lorengordon) [inline Makefile:401]: We can add this back now, including the targets we removed earlier for the Docker-based tools.

**Review by lorengordon — COMMENTED** [review]
This is looking really good now. A couple bits to cleanup, then will be good to go.

File: Makefile (hunk 4 above)
Comment (kumoy) [reply Makefile:401]: @lorengordon made the changes but I'm not sure if the naming conventions are correct..

**Review by lorengordon — APPROVED** [review]
awesome :shipit:

**Pull request merged**